source/index.md

```diff
@@ -13,74 +13,89 @@ Fortran <br> High-performance parallel programming language
 :::
 
 :::{div} sd-fs-3 sd-font-weight-bold sd-text-primary
-Features
+Why Fortran?
 :::
 
+Despite being thought by many outside the community to be an archaic (or dead) language, Fortran is very much alive. In fact, _because_ it is the oldest programming language in active use, it underpins multi-decadal computational efforts in Earth sciences, material sciences, engineering, physics and chemistry. 
```

Comment:
I would not start with "fortran is dead", that puts the reader in a negative "state", that we then need to get him out of. Rather, we want to stay positive.

I would say something like:

```suggestion
Fortran is the oldest programming language in active use and it underpins multi-decadal computational efforts in Earth sciences, material sciences, engineering, physics and chemistry. 
```